service/client.go

```diff
@@ -72,4 +72,14 @@ type Client interface {
 
 	// Returns whether we attached to a running process or not
 	AttachedToExistingProcess() bool
+
+	// Returns the address of a location expression
+	// loc ::= <filename>:<line> | <function>[:<line>] | /<regex>/
+	// * <filename> can be the full path of a file or just a suffix
+	// * <function> ::= <package>.<receiver type>.<name> | <package>.(*<receiver type>).<name> | <receiver type>.<name> | <package>.<name> | (*<receiver type>).<name> | <name>
+	// * <function> must be unambiguous
+	// * /<regex>/ will return a breakpoint for each function matched by regex
```

Comment:
Should we change the other commands? Because Some.Function could mean both the function Function in package Some and a regex matching Some1Function, Some2Function, etc…